helm/kube-bench-chart/templates/rbac.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
```

Comment:
There is a rolebinding but the role isn't defined IMO

Reply:
[I think it is](https://github.com/giantswarm/docs/blob/master/src/content/guides/securing-with-rbac-and-psp/index.md) but could be wrong?

Reply:
In the doc you linked it is also references: A binding is just that - a binding to the role.
The role still has to exist.